Services/CO.CDP.OrganisationInformation.Tests/Serialisation/LowerCamelCaseEnumConverterTest.cs

```diff
@@ -0,0 +1,44 @@
+using System.Text.Json;
+
+namespace CO.CDP.OrganisationInformation.Tests.Serialisation;
+
+public class LowerCamelCaseEnumConverterTest
+{
+    private readonly string _serialisedSinglePartyRoleString = "[\"supplier\"]";
+    private readonly string _serialisedMultiplePartyRolesString = "[\"supplier\",\"procuringEntity\"]";
+
+    private readonly List<PartyRole> _singlePartyRoleList = [PartyRole.Supplier];
+    private readonly List<PartyRole> _multiplePartyRolesList = [PartyRole.Supplier, PartyRole.ProcuringEntity];
+
+    [Fact]
+    public void Serialize_ShouldReadSinglePartyRoleEnum_WhenProvidedSerialisedLowerCamelCasedPartyRoles()
+    {
+        var jsonString = JsonSerializer.Serialize(_singlePartyRoleList);
+
+        Assert.Equal(_serialisedSinglePartyRoleString, jsonString);
```

Comment:
Use fluent assertions for consistency.

Also, can we just inline examples in each test case? I know it's duplication, but good duplication. It improves test readability.

Reply:
done.